### PR TITLE
Renames in PTQ config

### PR DESCRIPTION
**Pull request by janekl**
Overview:
* ~~Adding a workaround for MPI issues on Slurm for the PTQ stage~~
* Removing llama2 references as the config should support different families

### CODE REVIEW

File: launcher_scripts/tests/unit_tests/config_tests/test_main_config.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def test_config(self):
           - rlhf_rm: gpt3/2b_rm
           - rlhf_ppo: gpt3/2b_ppo
           - steerlm_reg : ac_sft/gpt_sft #rw_sft/training_rm
-          - ptq: llama2/quantization
+          - ptq: model/quantization
```
Comment (janekl) [inline launcher_scripts/tests/unit_tests/config_tests/test_main_config.py:29]: I'd prefer now to use a generic "model" name as this single config supports multiple model families (by specifying parameters appropriately).

Comment (thomasdhc) [reply launcher_scripts/tests/unit_tests/config_tests/test_main_config.py:29]: Gotcha, is there a need to keep `model/`? Could we just do `ptq: quantization`

Comment (janekl) [reply launcher_scripts/tests/unit_tests/config_tests/test_main_config.py:29]: Yeah I also wondered about it but this line https://github.com/NVIDIA/NeMo-Framework-Launcher/blob/main/launcher_scripts/nemo_launcher/core/stages.py#L477 isn't going to work.

We assume a path like `a/b` here, you can also see this pattern here: https://github.com/NVIDIA/NeMo-Framework-Launcher/blob/main/launcher_scripts/conf/config.yaml#L4-L22

Comment (thomasdhc) [reply launcher_scripts/tests/unit_tests/config_tests/test_main_config.py:29]: Hm I see! Would the quantization module work with all models?

Comment (janekl) [reply launcher_scripts/tests/unit_tests/config_tests/test_main_config.py:29]: That is the plan. It currently covers several models specified by `decoder_type` parameter here https://github.com/janekl/NeMo-Framework-Launcher/blob/jlasek/ptq_launcher_patch/launcher_scripts/conf/ptq/model/quantization.yaml#L42.

I just thought it's better to prepare a single config for maintenance & clarity reasons. I'm about to add a support matrix in Nemo docs to specify which models are supported. Also, there will be an assert in the code on decoder type.

Comment (janekl) [reply launcher_scripts/tests/unit_tests/config_tests/test_main_config.py:29]: Then on new models added we'll just update docs & assert, and no more changes in Launcher will be needed (like making a copy of new_model/quantization.py config).

**Review by thomasdhc — APPROVED** [review]
LGTM

**Pull request merged**